tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -59,7 +59,7 @@ def f():
 
     # Check that the warning is correct
     warning_info = [*errors.error_info_map.values()][0][0]
-    assert warning_info.symbol  == 'x'
+    assert warning_info.symbol in ('x', "'x'")
```

Comment:
The symbol is a string which could be quoted?

Reply:
When tests are run in developer-mode we use `repr` instead of `str` to get more information about the objects causing the errors. `str(PyccelSymbol('x'))` gives `'x'` but `repr(PyccelSymbol('x'))` gives `"'x'"`